src/Resources/config/services.yml

```diff
@@ -9,7 +9,7 @@ services:
         arguments:
             - "@service_container"
         tags:
-            - { name: kernel.event_listener, event: kernel.controller, method: onKernelController, priority: 254 }
+            - { name: kernel.event_listener, event: kernel.controller, method: onKernelRequest, priority: 31 }
```

Comment:
Should we probably set the priority of the `InitializeSystemListener` to 28 and the one of the `ContainerScopeListener` to 30 then? In case anyone needs to add a custom listener in between any of them.